packages/core/src/repositories/evaluationResultsRepository/index.ts

```diff
@@ -78,7 +78,7 @@ export class EvaluationResultsRepository extends Repository<
       .leftJoin(
         runErrors,
         and(
-          eq(runErrors.errorableId, evaluationResults.id),
+          eq(runErrors.errorableUuid, evaluationResults.uuid),
```

Comment:
This is critical change. But not so critical because there are 0 errors in DB now